source/GameInterface/Services/GameDebug/Interfaces/GameDebugInterface.cs

```diff
@@ -30,10 +35,10 @@ public void LoadDebugGame()
             GameLoopRunner.RunOnMainThread(InternalLoadDebugGame, false);
         }
 
-        private static readonly PropertyInfo FileDriver_SavePath = typeof(FileDriver).GetProperty("SavePath", BindingFlags.NonPublic | BindingFlags.Static);
-        private static readonly PlatformDirectoryPath SaveDir = (PlatformDirectoryPath)FileDriver_SavePath?.GetValue(null);
-        private static readonly PlatformFilePath SavePath = new PlatformFilePath(SaveDir, $"{LOAD_GAME}.sav");
-        private static readonly string FullSavePath = TaleWorlds.Library.Common.PlatformFileHelper?.GetFileFullPath(SavePath);
+        private static PropertyInfo FileDriver_SavePath => typeof(FileDriver).GetProperty("SavePath", BindingFlags.NonPublic | BindingFlags.Static);
```

Comment:
I assume the right-hand side is evaluated every time an expression-bodied property is accessed? Not that it matters much here given the context.

Reply:
The reason I changed this was to reduce memory overhead. May be negligible though